### ISSUE TITLE
TypeScript problem in graphql.d.ts

### ISSUE THREAD

**Issue by adriatic** [adriatic/TypeScript-problem#1]
I reported this https://github.com/redwoodjs/redwood/issues/6078 issue, discovered in the context of trying to create a TypeScript version of the Redwood Tutorial. At that time I was not aware of the same effort by @rushabhhere. As I was having the similar issue with a different application ([redwood-stripe](https://github.com/generalui/redwood-stripe)) I decided to convert the redwood tutorial app to typescript hoping to better understand my original problem, described below

---

In order  to reproduce my issue create any TypeScript app

`yarn create redwood-app --ts ./myapp`

Observe the just generated file `graphql.d.ts` in `api/types` folder, the definition of the `scalars` type

```
/** All built-in and custom scalars, mapped to their actual values */
export type Scalars = {
  ID: string;
  String: string;
  Boolean: boolean;
  Int: number;
  Float: number;
  BigInt: number;
  Date: string;
  DateTime: string;
  JSON: Prisma.JsonValue;
  JSONObject: Prisma.JsonObject;
  Time: string;
};
```

This same code rendered by VSCode is:

<img width="356" alt="image" src="https://user-images.githubusercontent.com/2712405/182034836-be463c77-b1a6-4deb-a0a2-1c0c30b819f5.png">

The issue with JSON and JSONObject is

<img width="434" alt="image" src="https://user-images.githubusercontent.com/2712405/182034915-bf4c3c9d-7feb-44de-b2eb-f0d887f5579a.png">

Seemingly an `import` statement is missing 😺

**Comment by Tobbe** [comment]
Yeah, this is not a problem with any particular RW project. If we should/could fix it, it'd be in the framework code for type generation. I'm not sure what change needs to be made though. If you could dig into it and find a solution that'd be awesome :)

**Comment by adriatic** [comment]
@Tobbe - I believe that I did find the change and it is trivial - please check my [explanation](https://github.com/redwoodjs/redwood-tutorial/pull/71#issuecomment-1200497286) to @rushabhhere (https://github.com/rushabhhere) so I do not pollute all discussions with my comments 😸 .
